app/assets/stylesheets/earthworks.css

```diff
@@ -152,3 +152,37 @@ Uncomment lines 142-144
     display: none;
   }
 }
+
+.filter .btn-outline-secondary {
+  --bs-btn-border-color: var(--stanford-fog-light);
+  --bs-btn-color: var(--stanford-black);
+  background-color: var(--stanford-fog-light);
+  --bs-btn-bg-color: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-black);
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-active-border-color: var(--stanford-fog-light);
+  --bs-btn-active-color: var(--stanford-black);
+  --bs-btn-padding-y: .25rem;
+}
+
+.applied-filter .remove:hover {
+  --bs-btn-hover-border-color: var(--stanford-fog-light);
+  --bs-btn-hover-bg: var(--stanford-fog-light);
+  --bs-btn-hover-color: var(--stanford-digital-red);
+  text-decoration: none;
+}
+
+.filter .constraint-value {
+  border-radius: 99px 0px 0px 99px;
+  padding-right: .25rem;
+}
+
+.filter .remove {
+  border-radius: 0px 99px 99px 0px;
+}
+
+.constraint-value, .remove {
```

Comment:
Can you explain why we need this rule.  I don't see any difference before and after applying this.

Reply:
I isn't centered when you don't apply this. The text is slightly closer to the top then the bottom.

Reply:
Can this be pushed up to Blacklight?  It seems to me this affects every Blacklight site, right?

Reply:
Actually, it doesn't affect demo.projectblacklight.org, so where is this difference coming from?